external/source/exploits/CVE-2023-29336/CVE-2023-29336/unused_code.ruleset

```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I don't think this file is necessary :)

Reply:
Yep I'm aware there are extra files, this was more done as a backup of current work.